OpenTabletDriver.Configurations/Parsers/Wacom/IntuosV2/IntuosV2WirelessReport.cs

```diff
@@ -0,0 +1,47 @@
+using System.Numerics;
+using System.Runtime.CompilerServices;
+using OpenTabletDriver.Tablet;
+
+namespace OpenTabletDriver.Configurations.Parsers.Wacom.IntuosV2
+{
+    public struct IntuosV2WirelessReport : ITabletReport, IHoverReport, IConfidenceReport, ITiltReport, IEraserReport
+    {
+        public IntuosV2WirelessReport(byte[] report)
+        {
+            Raw = report;
+
+            Position = new Vector2
+            {
+                X = Unsafe.ReadUnaligned<ushort>(ref report[2]) | (report[3] << 8),
+                Y = Unsafe.ReadUnaligned<ushort>(ref report[4]) | (report[5] << 8)
```

Comment:
Remove redundant bit operation.

```suggestion
                X = Unsafe.ReadUnaligned<ushort>(ref report[2]),
                Y = Unsafe.ReadUnaligned<ushort>(ref report[4])
```